tools/flaky-test-reporter/jsonreport/jsonreport.go

```diff
@@ -110,6 +111,23 @@ func GetReportForRepo(repo string, buildID int) (*Report, error) {
 	return report, nil
 }
 
+//GetReposWithReports gets all the repositories that are currently being reported on
+func GetReposWithReports() ([]string, error) {
+	job := prow.NewJob(jobName, prow.PeriodicJob, "", 0)
+	buildNum, err := job.GetLatestBuildNumber()
+	if err != nil {
+		return nil, err
+	}
+	build := job.NewBuild(buildNum)
+	artifactPaths := build.GetArtifacts()
+	var repos []string
+	for _, path := range artifactPaths {
+		splitPath := strings.Split(path, "/")
```

Comment:
why are we looking at all artifacts here?

Reply:
the reporter writes artifacts to `artifacts/<repo>/<artifact files>`, and in my testing GetArtifacts() only returns the paths one level deep in the artifacts dir, i.e. `artifacts/<repo>`. I added this directory structure in #925 to stop test suites from overwriting each other.

Reply:
This is actually a bug, GetArtifacts should return all files instead, I can fix it. You should assume it's returning all files for now

Reply:
https://github.com/knative/test-infra/pull/1110

Reply:
My point was there may be artifacts other than the ones added by the jsonreporter. For eg prow adds an empty junit_xml as well. We should be checking only the ones we are interested in.